ecosystem/sep-00xx.md

```diff
@@ -0,0 +1,131 @@
+## Preamble
+
+```
+SEP: 00xx
+Title: Stellar Web Authorization
```

Comment:
A slightly more descriptive title might be useful, like `Account Control Proofs` or something.